### ISSUE TITLE
Current master influx asks for an InfluxQL  for the version flag

### ISSUE THREAD

**Issue by fntlnz** [influxdata/influxdb#10451]
__System info: commit 2aa27686770a8d4782d638854165819695184bdc__

__Steps to reproduce:__

1. Clone the current master
2. Execute `go run cmd/influx/main.go -version` and note the wrong behavior
3. Checkout a released tag, like 1.6.4 - `git checkout tags/v1.6.4`
4. Execute `go run cmd/influx/main.go -version` and note the right behavior

__Expected behavior:__ [What you expected to happen]

Only version is reported
```
go run cmd/influx/main.go -version
InfluxDB shell version: unknown
```

__Actual behavior:__ [What actually happened]

Also the string `Enter an InfluxQL query` is reported.
```
go run cmd/influx/main.go -version
InfluxDB shell version: unknown
Enter an InfluxQL query
```

__Additional Information__:
I noted this behavior in a nightly build first

```
influx -version
InfluxDB shell version: 1.7.0~n201811020800
Enter an InfluxQL query
```

Then I've been able to reproduce it against the current master.

**Comment by fntlnz** [comment]
This still happens in the released 1.7.0 version, I downloaded the binaries from here, https://portal.influxdata.com/downloads

```
./influx -version                                                             InfluxDB shell version: 1.7.0
Enter an InfluxQL query
```

**Comment by fntlnz** [comment]
This is still there in 1.7.2

```
Connected to http://localhost:8086 version 1.7.2
InfluxDB shell version: 1.7.2
Enter an InfluxQL query
```

@rbetts wdyt?

**Comment by diraol** [comment]
Yeah, such a not nice behavior.
It makes the test automation a little bit harder than what is needed.

**Comment by diraol** [comment]
Btw, here is the "error source":
https://github.com/influxdata/influxdb/blob/1.7/cmd/influx/cli/cli.go#L1165

**Comment by stuartcarnie** [comment]
Closed by #13696 and #13697
